### PR TITLE
Added AI Chatbot

### LINKED ISSUE #1790: 💡[Feature]: Adding an AI bot
## Description
An AI chatbot can be extremely helpful and even necessary for an eCommerce website like VigyBag for several reasons:

- Chatbots can provide instant support to customers at any time, addressing common questions about products, shipping, returns, and more.
- Immediate answers to customer inquiries improve user experience.
- It can inform users about ongoing promotions, discounts, and new arrivals.
- It can provide detailed information about the materials, manufacturing processes, and environmental impact of products.
- It can guide users through the checkout process, ensuring they complete their purchases smoothly.

## Screenshots

Screenshot 2024-05-21 013906

Screenshot 2024-06-17 164703

### PR DESCRIPTION
**Pull request by antriksh-9**
## Fixes Issue
Closes #1790.

## Description
An AI chatbot can be extremely helpful and even necessary for an eCommerce website like VigyBag.
Users can receive personalized, interactive, and informative guidance regarding the products, shipping and returns etc.
It improves overall user experience and make websites more interactive and engaging.

## Video

https://github.com/user-attachments/assets/15bdfa85-b0b3-49a4-bc62-7580b0fc0548

## Summary by CodeRabbit

- **New Features**
	- Added web chat functionality to the app.
  
- **Style**
	- Adjusted the position of the "Go to Top" button on the screen for better accessibility.

### CODE REVIEW

**Comment by codervivek5** [comment]
Perfect 👍 

Can you change this logo with current logo

File: src/App.jsx [hunk 1]
```diff
@@ -53,9 +53,15 @@ import AdminPanel from "./Admin/Admin_Panel";
...
 import VigyForm from "./Admin/VigyForm";
+import { Helmet } from "react-helmet";
 
 export default function App() {
   return (
+    <>
+    <Helmet>
+    <script src="https://cdn.botpress.cloud/webchat/v2/inject.js"></script>
+    <script src="https://mediafiles.botpress.cloud/f910a06e-f7d4-4424-8642-e3cdffe933b5/webchat/v2/config.js"></script>
+    </Helmet>
```
[the remark this answers is not in the document]
Comment (codervivek5) [reply src/App.jsx:64]: kindly check this code

**Comment by codervivek5** [comment]
let me know after making these changes
-  add this chatbot above to the top icon button not below

**Comment by antriksh-9** [comment]
> Perfect 👍
> 
> Can you change this logo with current logo

Done.

**Comment by antriksh-9** [comment]
> let me know after making these changes
> 
> * add this chatbot above to the top icon button not below

When the user first arrives at the website the image below will be displayed on their screen. But if we will place the chatbot above the top button then the overall proportion of the website will be disturbed as initially the top button will be hidden.
What do you think?

Screenshot 2024-07-16 170841



that's the same thing I am saying to you suppose users land in the website and scroll more than half portion the top button will be displayed then if they want to use the chatbot top button will be hidden due to the chatbot if the being below of top button.

**Pull request merged**